deployment/k8s-config/harbor/opencadc/pvc-mount.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```

Comment:
Is this file for debugging the PVC?  Should it be in here?  If you think it should then its use should be documented.

Reply:
I've moved this file to deployment/k8s-config/scripts directory as it is a debugging tool.